src/components/content/Content.js

```diff
@@ -0,0 +1,63 @@
+import React, { Fragment } from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+// import Image from 'react-bootstrap/Image';
+
+import './Content.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function Content() {
+  return (
+   <Fragment>
+      <Container className="Content" fluid="true">
+         <Row className="Catchword">
+            <Col>
+               <h1>Probably the best team in the world</h1>
+               <p id="P1">We gathered the team to create some fancy app while doing the React Course.</p>
+               <p id="P2">Are we going to achieve that?</p>
+               <b>SURE WE DO!</b>
+            </Col>
+         </Row>
+      </Container>
+      <Container className="Content2" fluid="true">
```

Comment:
Team Members robi ktoś inny, nie ma sensu robić placeholderów bo to generuje później konflikty w gicie